modules/nw-ovn-k-anp-troubeshooting.adoc

```diff
@@ -0,0 +1,37 @@
+//module included in the following assemblies:
+//
+//networking/network_security/AdminNetworkPolicy/nw-ovn-k-anp-troubleshooting.adoc
+:_mod-docs-content-type: REFRENCE
+[id="anp-troubleshooting_{context}"]
+= Checking creation of ANP
+
+To check that your `AdminNetworkPolicy` (ANP) and `BaselineAdminNetworkPolicy` (BANP) are created correctly, check the status outputs of the following commands: `kubectl get anp` or `kubectl get banp`.
+
+A good status will indicate that `OVN DB plumbing was successful` and that `SetupSucceeded`
+
+.Example ANP with a good status
+[source,terminal]
+----
+...
+Conditions:
+Last Transition Time:  2024-06-08T20:29:00Z
+Message:               Setting up OVN DB plumbing was successful
+Reason:                SetupSucceeded
+Status:                True
+Type:                  Ready-In-Zone-ovn-control-plane Last Transition Time:  2024-06-08T20:29:00Z
+Message:               Setting up OVN DB plumbing was successful
+Reason:                SetupSucceeded
+Status:                True
+Type:                  Ready-In-Zone-ovn-worker
+Last Transition Time:  2024-06-08T20:29:00Z
+Message:               Setting up OVN DB plumbing was successful
+Reason:                SetupSucceeded
+Status:                True
+Type:                  Ready-In-Zone-ovn-worker2
+...
+----
```

Comment:
You suggest using `kubectl get` but the output on the example is from `kubectl describe`. It should be consistent (either provide some YAML output or change the suggestion to use describe)

In addition, it might be best to suggest using `oc` instead of `kubectl`. I know that `oc` is "`kubectl` plus more things" and that it behaves as `kubectl` if invoked as such, but for OCP documentation, it may be best to stick to use `oc` in the examples (in upstream docs, on the other hand, people have to use `kubectl` because `oc` is OCP-specific, but that's not our case here)

Reply:
I changed this to `oc describe anp` and `oc describe banp` that's what I used. I think this was just a mixup between writing from upstream and working on a live cluster.